Broken after upgrading to Python 3.8

Version: 0.1.1.dev192

Possibly related to changes to AST in 3.8

See also: https://github.com/pallets/werkzeug/issues/1551
Their fix is to change `ast.Module` to `ast.parse`: https://github.com/pallets/werkzeug/pull/1552/files

```
tests/test_worklist.py:46: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/test_worklist.py:29: in get_simplified_ircfg
    machine = Machine(cont.arch)
.env/lib/python3.8/site-packages/miasm/analysis/machine.py:126: in __init__
    from miasm.arch.x86 import jit
.env/lib/python3.8/site-packages/miasm/arch/x86/jit.py:5: in <module>
    from miasm.arch.x86.sem import ir_x86_16, ir_x86_32, ir_x86_64
.env/lib/python3.8/site-packages/miasm/arch/x86/sem.py:429: in <module>
    def xchg(arg1, arg2):
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <miasm.core.sembuilder.SemBuilder object at 0x7ff32d7d8ca0>, func = <function xchg at 0x7ff32d7e1e50>

    def parse(self, func):
        """Function decorator, returning a correct method from a pseudo-Python
        one"""
    
        # Get the function AST
        parsed = ast.parse(inspect.getsource(func))
        fc_ast = parsed.body[0]
        argument_names = [get_arg_name(name) for name in fc_ast.args.args]
    
        # Init local cache
        self._local_ctx = {}
    
        # Translate (blocks[0][0] is the current instr)
        blocks, body = self._parse_body(fc_ast.body, argument_names)
    
        # Build the new function
        fc_ast.args.args[0:0] = [
            gen_arg('ir', ast.Param()),
            gen_arg('instr', ast.Param())
        ]
        cur_instr = blocks[0][0]
        if len(blocks[-1][0]) == 0:
            ## Last block can be empty
            blocks.pop()
        other_blocks = blocks[1:]
        body.append(ast.Return(value=ast.Tuple(elts=[ast.List(elts=cur_instr,
                                                              ctx=ast.Load()),
                                                     ast.List(elts=other_blocks,
                                                              ctx=ast.Load())],
                                               ctx=ast.Load())))
    
        ret = ast.Module([ast.FunctionDef(name=fc_ast.name,
                                          args=fc_ast.args,
                                          body=body,
                                          decorator_list=[])])
    
        # To display the generated function, use codegen.to_source
        # codegen: https://github.com/andreif/codegen
    
        # Compile according to the context
        fixed = ast.fix_missing_locations(ret)
>       codeobj = compile(fixed, '<string>', 'exec')
E       TypeError: required field "type_ignores" missing from Module

.env/lib/python3.8/site-packages/miasm/core/sembuilder.py:349: TypeError
```

--- a/miasm/core/sembuilder.py
+++ b/miasm/core/sembuilder.py
@@ -336,10 +336,11 @@ class SemBuilder(object):
                                                               ctx=ast.Load())],
                                                ctx=ast.Load())))
 
-        ret = ast.Module([ast.FunctionDef(name=fc_ast.name,
-                                          args=fc_ast.args,
-                                          body=body,
-                                          decorator_list=[])])
+        ret = ast.parse('')
+        ret.body = [ast.FunctionDef(name=fc_ast.name,
+                                    args=fc_ast.args,
+                                    body=body,
+                                    decorator_list=[])]
 
         # To display the generated function, use codegen.to_source
         # codegen: https://github.com/andreif/codegen

